.drevops/docs/content/tools/twig-cs-fixer.md

```diff
@@ -0,0 +1,96 @@
+# Twig CS Fixer
+
+https://github.com/VincentLanglet/Twig-CS-Fixer
+
+> The missing checkstyle for twig!
+>
+> Twig CS Fixer aims to be what phpcs is to php. It checks your codebase for violations on coding standards.
+
+DrevOps comes with [pre-configured Twig-cs-fixer ruleset](../../../../.twig-cs-fixer.php) for Drupal projects.
+
+## Usage
+
+```shell
+vendor/bin/twig-cs-fixer
+```
+or
+```shell
+ahoy lint-be
+```
+
+## Configuration
+
+See [configuration reference](https://github.com/VincentLanglet/Twig-CS-Fixer/blob/main/docs/configuration.md).
+
+All global configuration takes place in the [`.twig-cs-fixer.php`](../../../../.twig-cs-fixer.php) file.
+
+Targets include custom modules and themes.
+
+Adding or removing targets:
+```php  hl_lines="12"
+$ruleset = new TwigCsFixer\Ruleset\Ruleset();
+$ruleset->addStandard(new TwigCsFixer\Standard\Twig());
+
+$finder = new TwigCsFixer\File\Finder();
+$finder->in(__DIR__ . '/web/modules/custom');
+$finder->in(__DIR__ . '/web/themes/custom');
+
+$config = new TwigCsFixer\Config\Config();
+$config->setRuleset($ruleset);
+$config->setFinder($finder);
+
+return $config;
+```
+
+## Ignoring
+
+All errors have an identifier with the syntax: `A.B:C:D` with
```

Comment:
@tannguyen04 
The point of this ticket was not to copy/paste from the source, but to retain the structure and put the specific items under headings - see how other tools doing it.

Reply:
See https://docs.drevops.com/1.19.0/tools/phpcs/#Configuration and headings